frame/support/src/traits/tokens/nonfungible_v2.rs

```diff
@@ -170,10 +170,14 @@ pub trait Mutate<AccountId, ItemConfig>: Inspect<AccountId> {
 	}
 }
 
-/// Trait for transferring a non-fungible item.
+/// Trait for transferring and locking/unlocking non-fungible sets of items.
 pub trait Transfer<AccountId>: Inspect<AccountId> {
 	/// Transfer `item` into `destination` account.
 	fn transfer(item: &Self::ItemId, destination: &AccountId) -> DispatchResult;
+	/// Disable the `item` transfer.
+	fn lock(item: &Self::ItemId) -> DispatchResult;
```

Comment:
q: should this be consistent with that of `fungible` api and have default empty implementations? 
Also, should this then be introduced to nonfungible v1?

Reply:
Another one - this should probably mention (and close?) https://github.com/paritytech/substrate/issues/13253 and have extra reviewers.

Reply:
> and have default empty implementations? 

As you can see a little bit below, there is a default implementation for those methods:
`... -> DispatchResult { Err(TokenError::Unsupported.into()) }`

Reply:
I can add this into v1 as well, though I think it's better to do it in a separate PR